Runtime/Model/Breadcrumbs/BacktraceBreadcrumbs.cs

```diff
@@ -182,5 +182,10 @@ public void Update()
         {
             EventHandler.Update();
         }
+
+        public static bool CanStoreBreadcrumbs(UnityEngineLogLevel logLevel, BacktraceBreadcrumbType backtraceBreadcrumbsLevel)
+        {
+            return (ulong)backtraceBreadcrumbsLevel != 0 && (ulong)logLevel != 0;

```

Comment:
The enums could be extended by adding a `None` value, which equals to 0. Disabling the logging would be easier as well, no need to assign a `(UnityEngineLogLevel)0` casted value.

Reply:
If you want to extend an Enum with None value that is equal to zero, it might be a bad decision, since None is interpreted by Unity engine UI

Reply:
At the beginning, I thought it won't work, but it looks like Unity handles it correctly. I'm validating it now with older Unity versions.

Reply:
All right.